archive/old_README.md

```diff
@@ -0,0 +1,205 @@
+# Wastewater Genomic Analysis for SARS-CoV2 Lineage determination
+
+A collection of scripts that UPHL uses to analyze wastewater sequencing data.
+
+This repository provides a suite of bash and python scripts used for the analysis of wastewater sequencing data, with a primary focus on determining the SARS-CoV2 lineages and their abundance. Initially developed for usage by UPHL bioinformaticians, these scripts can also be used by other researchers interested in similar genomic analyses. 
+
+The bash scripts set up the required directory structure for analysis, run bioinformatics analysis, and generate submission folders for NCBI. The output of the bioinformatics analysis is an aggregated CSV file containing the lineage data. You will need to run the python script under section [Wastewater Lineage Data Aggregator](#wastewater-lineage-data-aggregator) to complete the workflow as this python script will aggregate lineage data from previous runs with the current sequencing run and generate a CSV file which can be further used for visualization purpose on Microreact or deeper data exploration.
```

Comment:
This should be saved in git history in case you don't want to keep the extra file.

Reply:
Yeah! I could actually remove it.